charts/juicefs-csi-driver/templates/configmap.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if ge .Values.image.tag "v0.24.0" }}
```

Comment:
能否避免用 tag 来断定是否具备 cm 的能力？而是在 values 里引入一个单独的键值，比如 `enableglobalConfig: true`。

主要担心下列情况扰乱功能：

* 客户或用户会擅自修改 tag
* 有时候会需要用 nightly 镜像

Reply:
引入一个字段又麻烦很多，如果用户降级还得考虑这个字段。。

- 用户修改 tag 是很正常的事啊，我们依靠用户填入的版本决定是否启用感觉比较合理。

Reply:
按照 helm 常见的管理方式，可以设计成这样：

```
globalConfig:
  enabled: true
  data: |
    ...
```

这样一来，所有的 `ge .Values.image.tag "v0.24.0"` 判断都可以简单用这个 enabled 字段来替换，其实会更方便和显式。

主要是已经有客户这边把 tag 改成 `registry.example.com/juicefs:juicefs-csi-driver-v0.23.6` 这种形式了（恐怕是为了复用 registry repository），这种 tag 恐怕不支持 semver 对比操作。

Reply:
改了下

```yaml
globalConfig:
  # Set to false to disable global config
  enabled: true

  # Set to true to schedule mount pod to node with via nodeSelector, rather than nodeName
  enableNodeSelector: false
  
  ....
```